core/src/main/java/hivemall/topicmodel/OnlineLDAModel.java

```diff
@@ -350,66 +348,80 @@ private float computeApproxBoundForMiniBatch() {
         float score = 0.f;
 
         // prepare
-        final float[] gammaSum = new float[_miniBatchSize];
-        for (int d = 0; d < gammaSum.length; d++) {
-            gammaSum[d] += MathUtils.sum(_gamma[d], _K);
+        float[] gammaSum = new float[_miniBatchSize];
+        Arrays.fill(gammaSum, 0.f);
```

Comment:
zero filling is not required in java. `gammaSum` has initialized with 0 for each entry.